Documentation/nfs.md

```diff
@@ -80,6 +80,57 @@ rook-nfs-provisioner-7b5ff479f6-688dm   1/1     Running   0          102s
 rook-nfs-webhook-74749cbd46-6jw2w       1/1     Running   0          102s
 ```
 
+## Create Openshift Security Context Constraints
+
+On Openshift cluster, before creating NFS Server we need to create the security context constraints needed by nfs-server pods which running on openshift. The following yaml is found in `scc.yaml` under `/cluster/examples/kubernetes/nfs`.
```

Comment:
I would also add language here to make it clear that folks ONLY need to do this for OpenShift and that they can skip to the next section if they aren't running OpenShift.  Consider adding a link that they can click and jump to the next section.

Suggested text:

On OpenShift clusters, we will need to create some additional security context constraints.  If you are **not** running in OpenShift you can skip this and go to the [next section](/FILL_IN_LINK_TO_NEXT_SECTION.md).

To create the security context constraints for nfs-server pods, we can use the following yaml, which is also found in `scc.yaml` under `/cluster/examples/kubernetes/nfs`.